### DISCUSSION TITLE
Is an empty initializer list valid C code?

### DISCUSSION THREAD

**Discussion by polluks** [cc65/cc65#1802] [Polls]
So far I found two lazy compilers and a strict one, where should cc65 go?
see https://stackoverflow.com/a/17589839/1430535
and https://stackoverflow.com/a/72836583/1430535

**Comment by mrdudz** [comment]
Would you please explain what the point is... or what the question is?

**Comment by rofl0r** [comment]
c99's universal initializer is `= {0}`, and a c23 proposal is to also allow `= {}` like in c++.

**Comment by mrdudz** [comment]
sure. and...?

**Reply by polluks** [reply]
Feel free to poll

**Comment by acqn** [comment]
Will cc65 support C23?

**Comment by mrdudz** [comment]
Same answer as before when it comes to standards: we want to support C89 and C99 in the first place. Additional stuff can be added to the "cc65" standard. I seriously doubt we will ever actually past anything after C99. Not in this decade anyway :)

(and please stop making polls for random things like this. its just noise.)
